cmd/iam-store.go

```diff
@@ -536,13 +546,13 @@ func (store *IAMStoreSys) LoadIAMCache(ctx context.Context) error {
 
 		bootstrapTraceMsg("loading STS users")
 		// load STS temp users
-		if err := store.loadUsers(ctx, stsUser, newCache.iamUsersMap); err != nil {
+		if err := store.loadUsers(ctx, stsUser, newCache.iamSTSAccountsMap); err != nil {
```

Comment:
Does this change fix an issue in master ?

Reply:
Yes, it is mentioned in the commit also:
> Also fix a bug where with etcd, STS accounts could get loaded into the
> iamUsersMap instead of the iamSTSAccountsMap.

The bug happens in etcd only - we will be deprecating it too, but this will fix the problem for now.